.github/workflows/reusable-performance.yml

```diff
@@ -109,6 +110,12 @@ jobs:
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           show-progress: ${{ runner.debug == '1' && 'true' || 'false' }}
+          fetch-depth: ${{ github.event_name == 'workflow_dispatch' && '2' || '1' }}
+
+      # The `workflow_dispatch` event is the only one missing the needed SHA to target.
```

Comment:
@joemcgill Is this sufficient enough to call out and [address your question](https://github.com/WordPress/wordpress-develop/pull/7058/files#r1683347160) for future maintainers?